packages/neutrino/src/neutrino.js

```diff
@@ -21,10 +21,17 @@ class Neutrino extends EventEmitter {
 
     this.config = new Config();
     this.options = merge(options, { root, source, output, tests, node_modules, entry });
+    this.useQueue = []
   }
 
   use(preset, options = {}) {
-    preset(this, options);
+    this.useQueue.push(() => preset(this, options));
+  }
+
+  load() {
+    return this.useQueue.reduce((result, task) => {
```

Comment:
I like the use of `reduce`, as well as the initial `Promise.resolve` to kick it off. Makes `use` leaner. :+1: 

To pass linting, change to:

```js
load() {
  return this.useQueue.reduce((result, task) => result.then(task), Promise.resolve());
}
```